src/Sentry/Protocol/Trace.cs

```diff
@@ -63,7 +78,8 @@ internal set
         Operation = Operation,
         Origin = Origin,
         Status = Status,
-        IsSampled = IsSampled
+        IsSampled = IsSampled,
+        _data = _data.ToDict()
```

Comment:
```suggestion
```

I think this is redundant right? It already happens here:
https://github.com/getsentry/sentry-dotnet/blob/eda625a36696264bd6751ada2003c061d22dc847/src/Sentry/Protocol/Trace.cs#L53

Reply:
It's a clone to ensure dictionary updates in the clone are not hitting the original collection.

Reply:
Argh... tricked by GitHub's review interface. It looked like this was in the SetData method:
<img width="649" alt="image" src="https://github.com/user-attachments/assets/2022e9a8-4b76-43e7-b739-425c606fe95d" />

I see it's actually in the `Clone` method!